src/main/main.ts

```diff
@@ -205,7 +205,17 @@ async function initializeApp() {
     if (app.commandLine.hasSwitch("v2")) win = await createV2Window();
     else win = await createWindow();
     createTray(path.join(app.getAppPath(), logoImage));
-    remoteNode = await initializeNode();
+    try {
+      remoteNode = await initializeNode();
+    } catch (e) {
```

Comment:
How about adding a log related to `e`? (i.e. `console.error(e)`)

Reply:
Nice catch. It would be better if the thrown error on `config.ts` included more information about the error.